### ISSUE TITLE
Unused declaration fix removes copyright

### ISSUE THREAD

**Issue by ajmathews** [cartant/tslint-etc#34]
If the file has a copyright on top and 'no-unused-declaration' fix is run, the fix deletes the copyright as well if the first import in the file requires to be fixed.

Example if `Subject` is not being used below
```
/**
 * @copyright Copyright abc
 */

import { Subject } from 'rxjs';
```

**Comment by cartant** [comment]
This should be fixed in `1.6.1`.

**Comment by ajmathews** [comment]
Seems to be fixed, thank you
